### DISCUSSION TITLE
Add Buisness Class seats

### DISCUSSION THREAD

**Discussion by CaptainJankez** [magknight/787aviatorsTracker#807] [Ideas]
My idea is to bring the business class to life in this plane, instead of full economy cabin, which is rarely ever seen in any dreamliner, I think that idea would fit well with your project to make it even a better one.

**Comment by Simleybob** [comment]
And bring back a way to customise the cabin per livery.

**Reply by aeroniemi** [reply]
you have the same amount of ability to customise the cabin as you always have?

**Reply by Simleybob** [reply]
I thought you removed that for the time being?
